core/src/main/java/org/elasticsearch/action/ingest/delete/DeletePipelineAction.java

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 
-package org.elasticsearch.plugin.ingest.transport.delete;
+package org.elasticsearch.action.ingest.delete;
```

Comment:
one more thing about packages: I have now o.e.ingest and o.e.ingest.core . The former is the es runtime, while the latter holds core things that may one day become a separate library, potentially independent from es core.